test/test_ecs_utils.py

```diff
@@ -6,7 +6,24 @@
 
 @pytest.fixture(scope='module')
 def ecs_utils():
-    return ECSUtils(cluster_name='dummy-cluster')
+    return ECSUtils()
+
+
+def mock_list_ecs_services(*, cluster):
+    """ Mock API signature """
+    ignored(cluster)
+    return {
+        'serviceArns':
+            [
+                'arn:aws:ecs:us-east-2:1234566777:service/c4-ecs-trial-alpha-stack-CGAPDockerCluster-Z4m1uYa2J11O/c4-ecs-trial-alpha-stack-CGAPIndexerService-YihcMquIc354',
+                'arn:aws:ecs:us-east-2:1234566777:service/c4-ecs-trial-alpha-stack-CGAPDockerCluster-Z4m1uYa2J11O/c4-ecs-trial-alpha-stack-CGAPDeploymentService-NRyGGBTSnqbQ',
+                'arn:aws:ecs:us-east-2:1234566777:service/c4-ecs-trial-alpha-stack-CGAPDockerCluster-Z4m1uYa2J11O/c4-ecs-trial-alpha-stack-CGAPIngesterService-QRcdjlE5ZJS1',
+                'arn:aws:ecs:us-east-2:1234566777:service/c4-ecs-trial-alpha-stack-CGAPDockerCluster-Z4m1uYa2J11O/c4-ecs-trial-alpha-stack-CGAPWSGIService-oDZbeVVWjZMq'
+            ],
+        'ResponseMetadata':
+            {'RequestId': 'not-a-uuid',
+             'HTTPStatusCode': 200}
```

Comment:
What is this telling me? The request id looks like an error but it's returning a 200. I'm not sure I fully get the context though.